### DISCUSSION TITLE
Grid Issue

### DISCUSSION THREAD

**Discussion by etonucci** [Stellarium/stellarium#3589] [General]
Hi All,
I noticed that the equatorial grid and altazimut grid disappear when I try to use the eyepices plug in and then the telrad pluging.
Infact if I select the equatorial grig and altazimut grid, then select an object for ex. Polaris, then I click on the eyepieces plugin and then the telrad plugin all the grids disappear and is not possible to see them again.
There is a solution?
Thanks

**Comment by alex-w** [comment]
Yes, please check Oculars plugin settings
